### ISSUE TITLE
Whisper Mate not able to get transcritions.

### ISSUE THREAD

**Issue by holanino** [Marksdo/Whisper#9]
https://tmpsend.com/l1QOyJkL

Whisper Mate Pro is not able to get the transcription. Specially 1:20 to 1:48 time. Also 2:11 to 3:15 and others.

Help

**Comment by Marksdo** [comment]
<img width="826" alt="image" src="https://github.com/Marksdo/Whisper/assets/117704549/03a22335-b4f5-44e9-8f87-133c283015f1">
<img width="751" alt="image" src="https://github.com/Marksdo/Whisper/assets/117704549/e720b8e8-34a7-4041-9a0d-f23ffc35de33">

Which model do you use? i can get the result as you mentioned.

**Comment by Marksdo** [comment]
upper is using the tiny model
below use the medium.en model
<img width="1409" alt="image" src="https://github.com/Marksdo/Whisper/assets/117704549/80ab3b76-6e4f-49de-903c-f4098f5008d2">
<img width="1409" alt="image" src="https://github.com/Marksdo/Whisper/assets/117704549/28d24a58-9a33-474c-8228-88d306d0dbe7">


upper result using these model options
<img width="364" alt="image" src="https://github.com/Marksdo/Whisper/assets/117704549/9d260135-1bff-46ca-b14b-71268149753d">

**Comment by Marksdo** [comment]
Please use the new V4.5.1 in app store and retry. I add an ai arguments with template. Try use difference template arguments to process. Thanks for info me this issue to make me know this area argument is not clear.

<img width="364" alt="image" src="https://github.com/Marksdo/Whisper/assets/117704549/45485e36-6872-4fe3-beb6-c9990e7ff523">

**Comment by holanino** [comment]
Yes it works much better, thank you.  It works very slow now, takes a long time.
